analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -177,6 +173,52 @@ public class Bar
             VerifyAllInstructions(exit);
         }
 
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_ExtremelyNestedExpression_NotSupported_FromExpression()
```

Comment:
both `Cfg_ExtremelyNestedExpression_NotSupported_FromExpression` and `Cfg_ExtremelyNestedExpression_NotSupported_FromBodyMethod` are from body methods...

You could rename 
```suggestion
        public void Cfg_ExtremelyNestedExpression_NotSupported_FromAssignment()
```

Reply:
Good catch! I actually wanted to check `FromExpression`. My intention is to cover all branches here:
https://github.com/SonarSource/sonar-dotnet/blob/84c8cde333395a1f13e27f95832b9e1937c8721c/analyzers/src/SonarAnalyzer.CFG/ControlFlowGraph/CSharpControlFlowGraphBuilder.cs#L79-L90